<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zhukuang</class>
 <widget class="QWidget" name="zhukuang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/Set/1-drawshape_fx_bg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>20</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sp.qrc">:/tu/Res/Set/boundary.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>120</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sp.qrc">:/tu/Res/Set/boundary.png</pixmap>
   </property>
  </widget>
  <widget class="Gnj" name="gnj">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/gnj-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SZ" name="sz">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/sz-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="third_stack">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>140</y>
     <width>311</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="label_photograph">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_photograph { 
    border-image: url(:/tu/Res/pz.png); 
    border: none; 
    background: transparent; 
}
#label_photograph:pressed { 
    border-image: url(:/tu/Res/pz.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="label_video">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>60</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_video { 
    border-image: url(:/tu/Res/sx.png); 
    border: none; 
    background: transparent; 
}
#label_video:pressed { 
    border-image: url(:/tu/Res/sx.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_Awb">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#m_pBtn_Awb { 
    border-image: url(:/tu/Res/MainWindow/awb.png); 
    border: none; 
    background: transparent; 
}
#m_pBtn_Awb:pressed { 
    border-image: url(:/tu/Res/MainWindow/awb_sel.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>冻结画面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_Freeze">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#m_pBtn_Freeze { 
	border-image: url(:/tu/Res/free-1.png);
    border: none; 
    background: transparent; 
}
#m_pBtn_Freeze:pressed { 
   	border-image: url(:/tu/Res/free-2.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>AE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_AE">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/tu/Res/MainWindow/ae-1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>画面旋转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_Rot">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>350</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/tu/Res/MainWindow/rot.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>图像窗口形状</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_DrawShape">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/tu/Res/Set/drawshape.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>保存图片</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>保存视频</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>图片查看</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>视频查看</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seek_photo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#seek_photo { 
    border-image: url(:/tu/Res/Set/photoshow.png); 
    border: none; 
    background: transparent; 
}
#seek_photo:pressed { 
    border-image: url(:/tu/Res/Set/photoshow_sel.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="seek_video">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#seek_video { 
    border-image: url(:/tu/Res/Set/videoplay.png); 
    border: none; 
    background: transparent; 
}
#seek_video:pressed { 
    border-image: url(:/tu/Res/Set/videoplay_sel.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>时间设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_DateSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#m_pBtn_DateSet { 
    border-image: url(:/tu/Res/Set/dateset.png); 
    border: none; 
    background: transparent; 
}
#m_pBtn_DateSet:pressed { 
    border-image: url(:/tu/Res/Set/dateset_sel.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>语言设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_LanSet">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#m_pBtn_LanSet { 
    border-image: url(:/tu/Res/Set/lanset.png); 
    border: none; 
    background: transparent; 
}
#m_pBtn_LanSet:pressed { 
    border-image: url(:/tu/Res/Set/lanset_sel.png); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>格式化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_Format">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#m_pBtn_Format { 
    border-image: url(:/tu/Res/Set/format.png); 
    border: none; 
    background: transparent; 
}
#m_pBtn_Format:pressed { 
    border-image: url(:/tu/Res/Set/format_sel.png); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>重置设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_Reset">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>330</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#m_pBtn_Reset { 
    border-image: url(:/tu/Res/Set/reset.png); 
    border: none; 
    background: transparent; 
}
#m_pBtn_Reset:pressed { 
    border-image: url(:/tu/Res/Set/reset_sel.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_Upgrade">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#m_pBtn_Upgrade { 
    border-image: url(:/tu/Res/Set/upgrade.png); 
    border: none; 
    background: transparent; 
}
#m_pBtn_Upgrade:pressed { 
    border-image: url(:/tu/Res/Set/upgrade_sel.png); 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>版本信息</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="prompt_photo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);font-size: 32px;font: bold;</string>
   </property>
   <property name="text">
    <string>taking a photo</string>
   </property>
  </widget>
  <widget class="QLabel" name="video_time">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);font-size: 32px;font: bold;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="m_pWdg_Panel_DrawShape" native="true">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>540</y>
     <width>151</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QLabel" name="m_pLb_DrawShape_Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/tu/Res/Set/drawshape_title.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtn_DrawShape_FX">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/tu/Res/Set/drawshape_fx_sel.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtn_DrawShape_YX">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/tu/Res/Set/drawshape_yx.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtn_DrawShape_BJX">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/tu/Res/Set/drawshape_bjx.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/lg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLb_Date">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>400</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size: 25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>720</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/MainWindow/elec2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/usb_ts.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gnj</class>
   <extends>QLabel</extends>
   <header>zhukuang.h</header>
  </customwidget>
  <customwidget>
   <class>SZ</class>
   <extends>QLabel</extends>
   <header>zhukuang.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sp.qrc"/>
 </resources>
 <connections/>
</ui>
